logging/logging.go

```diff
@@ -15,8 +16,21 @@ const (
 	jsonLogFormat    = "json"
 )
 
-// CreateLogger create and configure zerolog logger based on app config.
-func CreateLogger(instanceName, format, level string, logOrigin bool) (*zerolog.Logger, error) {
+// GetDefaultLogger create and configure default zerolog logger. It should be used before config is loaded
+func GetDefaultLogger() zerolog.Logger {
+	logger, err := createLogger("spv-wallet-default", jsonLogFormat, "debug", true)
```

Comment:
Should we have `jsonLogFormat` here? I would vote for `consoleLogFormat` as a default 🤔

Reply:
From what I remember, we're setting it to json because of kibana, that when it's starting as standard log the collector was failing because it was expecting a json.

Reply:
Okay, it's clear for me now why it's different from defaults from config.example